javacutil/src/main/java/org/checkerframework/javacutil/CollectionUtils.java

```diff
@@ -23,4 +28,203 @@ protected boolean removeEldestEntry(Map.Entry<K, V> entry) {
       }
     };
   }
+
+  ///
+  /// Temporary utility methods
+  ///
+
+  // TODO: After code review, copy these methods into plume-util.
+  // After plume-util 1.6.6 is released, use these methods from it.
+
+  /**
+   * Returns a copy of {@code orig}, where each element of the result is a clone of the
+   * corresponding element of {@code orig}.
+   *
+   * @param <T> the type of elements of the collection
+   * @param <C> the type of the collection
+   * @param orig a collection
+   * @return a deep copy of {@code orig}
```

Comment:
I'm not sure if it would be better to call this method `deepCopyElements`. This part of the JavaDoc suggests that it would.

In general, though, there seems to be some confusion between the terms "clone" and "deepCopy" - maybe we should define them (perhaps in the `DeepCopyable` JavaDoc?).